containenv/commands/containenv.py

```diff
@@ -219,3 +234,41 @@ def write_runcontainenv(self):
     def publish_image(self):
         '''if there's a shared registry publish to it'''
         pass
+
+
+class Activate(object):
+    '''Usage: containenv activate [help]
```

Comment:
oops...  this breaks style checks...  and upon a moments reflection...   properly so...   I think rapcom has to handle this gracefully.